.travis.yml

```diff
@@ -38,7 +38,8 @@ script:
   - cargo test --features "$BINDGEN_FEATURES"
   - cargo test --release --features "$BINDGEN_FEATURES"
   - cd ../bindgen-integration
-  - cargo test
+  - cargo test --features "$BINDGEN_FEATURES"
+  - cargo test --release --features "$BINDGEN_FEATURES"
```

Comment:
I think this is getting big enough that we should pull it out into a `./ci/test` script. In a follow up, of course.